serialization/src/main/scala/sbt/serialization/CustomPicklerUnpickler.scala

```diff
@@ -1,18 +1,23 @@
 package sbt.serialization
 
-import sbt.serialization.pickler.{
+// TODO - Why are type aliases not workign?
```

Comment:
Hmm, why was it working before?  The only alteration is having two package objects instead of one.